oqs-template/generate.yml

```diff
@@ -1516,3 +1516,138 @@ sigs:
                     'pretty_name': 'ECDSA p521',
                     'oid': '1.3.9999.8.5.2',
                     'code_point': '0xfef5'}]
+  -
+    family: 'CROSS'
+    variants:
+      # RSDP 128
+      -
+        name: 'CROSSrsdp128balanced'
+        pretty_name: 'CROSS-rsdp-128-balanced'
+        oqs_meth: 'OQS_SIG_alg_cross_rsdp_128_balanced'
+        oid: '1.3.6.1.4.1.62245.2.1.1'
+        code_point: '0xfef6'
```

Comment:
Is it intentional that no hybrid forms ("mix_with") are specified?

Reply:
Would it be ok to address hybrids in a future PR? I'd like to make sure I fully understand how they are composed first.

Reply:
Of course!